GlobalExceptionFilter.App/HttpGlobalExceptionFilter.cs

```diff
@@ -51,11 +51,11 @@ public void OnException(ExceptionContext context)
 
         private void Log(ExceptionContext context)
         {
-            if (context.Exception.GetType() == typeof(CustomApiException))
+            if (context.Exception is CustomApiException customApiException)
             {
                 var exception = (CustomApiException)context.Exception;
 
-                if (exception.ProblemDetails.Status < 500 && exception.ProblemDetails.Status >= 400)
+                if (exception.ProblemDetails.Status < (int)HttpStatusCode.InternalServerError && exception.ProblemDetails.Status >= (int)HttpStatusCode.BadRequest)
```

Comment:
line 56 should be removed. Morover, line 58 should converted to "customApiException.ProblemDetails..."